Maskbook/Extension/Introspect+UITextField.swift

```diff
@@ -0,0 +1,53 @@
+import Introspect
+
+extension TargetViewSelector {
+    static func siblingContainingOrAncestorOrAncestorChild<TargetView: PlatformView>(from entry: PlatformView) -> TargetView? {
+        if let sibling: TargetView = siblingContaining(from: entry) {
+            return sibling
+        }
+        return Introspect.findAncestorOrAncestorChild(ofType: TargetView.self, from: entry)
+    }
+}
+
+extension Introspect {
+    static func findAncestorOrAncestorChild<AnyViewType: PlatformView>(ofType type: AnyViewType.Type, from entry: PlatformView) -> AnyViewType? {
+        var superview = entry.superview
+        while let s = superview {
+            if let typed = s as? AnyViewType ?? findChildUsingFrame(ofType: type, in: s, from: entry) {
```

Comment:
```suggestion
            if let typed = s as? AnyViewType ?? findChildUsingFrame(ofType: type,
                                                                    in: s,
                                                                    from: entry) {
```

Reply:
In this case wrap parameters is not a good option.